doc/modules/ROOT/pages/graph-object.adoc

```diff
@@ -135,6 +135,20 @@ This in particular means that:
 If multiple node dataframes are used, they need to contain distinct node ids across all node data frames.
 * Prior to the `construct` call, a call to `GraphDataScience.set_database` must have been made to explicitly specify which Neo4j database should be targeted.
 
+[[graph-object-load-cora]]
+=== Loading the CORA dataset
+
+For convenience, the library is shipped with the https://paperswithcode.com/dataset/cora[CORA dataset].
```

Comment:
https://paperswithcode.com/dataset/cora lists the license as "Unknown", which is a bit tricky, as (depending on the jurisdiction) it can be interpreted as being copy-righted (some SE thread for example: https://softwareengineering.stackexchange.com/questions/26548/what-is-the-default-software-license). I don't know what the standard in that area is, however the site above also has a "Dataset loaders" section, which looks like places where others have already included that dataset.

Reply:
Speaking of, can we now get included in that "Dataset loaders" section?

Reply:
Hmm good point. We'll need to investigate this further